### ISSUE TITLE
Material 2 Tabs doesn't work well with manual Change Detection Strategy

### ISSUE THREAD

**Issue by naveedahmed1** [angular/material2#1037]
Angualr2 allows us to implement local change detection using ChangeDetectorRef. Which is very useful when we know exactly when our data changes (or change occur). 

For example we can detach the Change Detector from a component using ChangeDetectorRef.detach() and when there is any change we can use ChangeDetectorRef.detectChanges() to manually run change detection.

This doesn't seems to work well with Material Tabs.

Please have a look at the below plunker:

http://plnkr.co/edit/PaZuholUTzMCpEPclUJw?p=preview

I detached the Change Detector in the constructor. Then in ngOnInit I set the data of Tab 1 and then call this.cdr.detectChanges(); to run change detection and as a result you can see below text in preview.

Tab 1: This is tab 1

The template of the Tab 1 is also bind to the tab1Data property but the data in the Tab 1 isn't updated.

**Comment by jelbourn** [comment]
This should be resolved now that portals are synchronous.
